frontend/packages/dev-console/src/components/import/git/GitSection.tsx

```diff
@@ -255,7 +255,6 @@ const GitSection: React.FC<GitSectionProps> = ({
 
       const gitService = getGitService(
         url,
-        // TODO: ODC-6250 - GitTypes is not compatibily to git service type GitProvider
         gitType as any,
```

Comment:
We can now remove the `as any` here, or?

```suggestion
        gitType,
```

Reply:
Thanks, I had missed this. Updated.